backends/tfhe-cuda-backend/cuda/tests_and_benchmarks/benchmarks/benchmark_pbs.cpp

```diff
@@ -362,9 +358,9 @@ MultiBitPBSBenchmarkGenerateParams(benchmark::internal::Benchmark *b) {
   // input_lwe_ciphertext_count
   std::vector<MultiBitPBSBenchmarkParams> params = {
       // 4_bits_multi_bit_group_2
-      (MultiBitPBSBenchmarkParams){818, 1, 2048, 22, 1, 1, 2, 0},
+      (MultiBitPBSBenchmarkParams){818, 1, 2048, 22, 1, 1, 2},
       // 4_bits_multi_bit_group_3
-      (MultiBitPBSBenchmarkParams){888, 1, 2048, 21, 1, 1, 3, 0},
+      (MultiBitPBSBenchmarkParams){888, 1, 2048, 21, 1, 1, 3},
   };
 
   // Add to the list of parameters to benchmark
```

Comment:
You need to update this piece of code as well to remove the loop on the lwe_chunk_size.